workspaces/simon-game/src/app/util/playNote.ts

```diff
@@ -0,0 +1,5 @@
+import { config } from "../constants.ts";
+import { playSound } from "./playSound.ts";
+
+export const playNote = (freq: number) =>
```

Comment:
per later comments, for consistency

```suggestion
export function playNote(freq: number): void {
```